script.js

```diff
@@ -1,53 +1,58 @@
-window.onload = () => {
-  let books = [];
+class Book {
+  static books = [];
+
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
 
-  const addBooksButton = document.querySelector('#add-book');
-  const bookTitle = document.querySelector('#book-title');
-  const bookAuthor = document.querySelector('#book-author');
+  addBook() {
+    Book.books.push(this);
+    Book.store(Book.books);
+  }
```

Comment:
- [ ] Let's check this function because it is not working as expected. It would be better to add some inputs validations as well to avoid submitting invalid values resulting in a null value.

![Screenshot from 2021-11-23 21-28-00](https://user-images.githubusercontent.com/40503704/143099023-f4af46ca-0bdd-4876-8b08-0a33de0a76cd.png)

Reply:
Done